packages/teleport/src/components/BannerList/useAlerts.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { useState, useEffect } from 'react';
+import Logger from 'shared/libs/logger';
+
+import { fetchClusterAlerts } from 'teleport/services/alerts';
+import useStickyClusterId from 'teleport/useStickyClusterId';
+
+import type { ClusterAlert } from 'teleport/services/alerts';
+
+const logger = Logger.create('ClusterAlerts');
+
+const DISABLED_BANNERS = 'disabledAlerts';
+
+function addHours(date: number, hours: number) {
+  return date + hours * 60 * 60 * 1000;
```

Comment:
can we make the 60 * 60 * 1000 a const? b/c i'm not sure what it means

Reply:
Sure.

There are 3,600,000ms in an hour and this is just a common way of writing that.